README.md

```diff
@@ -264,10 +261,6 @@ spec:
                    value: "helios"
                  - name: UID
                    value: "1000"
-                 - name: GID
-                   value: "1000"
-                 - name: PASSWORD
-                   value: "password"
```

Comment:
What doe these changes have to do with idle detection?

Reply:
This is just to keep the docs more in line. The examples typically only include required env-vars and I was updating the docs to reflect the new IDLE_TIME var. I saw that these were no longer needed either so fixed up those docs while I was there.